### ISSUE TITLE
AGORA Deseja poder editar estes dados

### ISSUE THREAD

**Issue by pedrojovelli** [pedrojovelli/AgoraPartnerships#144]
PÁGINA:
http://agora2030.org/entrepreneurs/agora-growth-bootcamps/

ícones e textos (inglês e espanhol)

Se possível montar via spreadsheet para que eles possam mudar a ordem se quiserem.

![screen shot 2018-11-07 at 15 17 10](https://user-images.githubusercontent.com/15367213/48148151-40867b00-e2a0-11e8-9956-18f83ed0cb54.png)

**Comment by robanquieri** [comment]
Pedro,

Finalizei a implementação baseada em planilha do Google. A planilha é esta:

https://docs.google.com/spreadsheets/d/1ux1IyoAPxxYIWYgAvc4j3J5ms-JgKIRQmMHRy8_3Z5w

A aba de numero 1 se refere a página Agora Growth Bootcamps, a 2 Agora Chile e a 3 Agora Nicarágua.

Eu sugiro que você crie um PDF com os ícones e respectivos nomes, os ícones que podem ser utilizados estão nesta url https://drive.google.com/file/d/1UTjEK8ZrKh-vQXV_j0NZGcjxv59zqEUJ/view?usp=sharing

**OBS: Na planilha, no campo imagem, é somente o nome, sem a extensão do ícone. A ordem das abas não pode ser alterada de forma alguma!**

**Comment by pedrojovelli** [comment]
Caso queiram subir ícones diferentes, devem subir em algum diretório no FTP ?

**Comment by robanquieri** [comment]
Sim, no caminho "/public_html/wp-content/themes/agora2018/assets/images/icons"
